### DISCUSSION TITLE
order of DIV ops

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#282] [Answers & Questions] [answered]
how do i control the order of multiple div ops to create something like this ?

```
<div id=”div-container1”>
Inhalt
</div>

<div id=”div-container2”>
Inhalt
</div>
```

**Comment by wirmachenbunt** [comment, marked as answer]
maybe like this 
<img width="241" alt="image" src="https://user-images.githubusercontent.com/1138673/175290470-7b8e8ede-bc2d-47de-9a67-ca6d71be5a19.png">

**Reply by steam0r** [reply]
you can also use a container div and then ElementChilds

**Reply by wirmachenbunt** [reply]
aha, that was what i was looking for. cheers !
